src/API/Management/Blacklists.php

```diff
@@ -15,7 +15,11 @@ public function getAll($aud)
     {
         $response = $this->httpClient->get('/blacklists/tokens?'.http_build_query(['aud' => $aud]));
 
-        return ResponseMediator::getContent($response);
+        if (200 === $response->getStatusCode()) {
```

Comment:
depends how do you see it. 
I think it converts the HTTP Client Responses (which in this case are compliant with PSR7 Responses) to the proper SDK response. 

204 => success (should do nothing)
204 => success, check for content type and parse if needed
4xx or 5xx => failed 💥, check for content type and parse if needed
something else => failed, check for content type and parse if needed

Anyway, if you prefer, you can move the mapping to a different entity and I think we can make it generic enough (following the previous mapping) to work with any endpoint instead of doing it in a per endpoint basis.